configs/pie-components-config/playwright-lit-visual-config.js

```diff
@@ -17,7 +17,7 @@ export function getPlaywrightVisualConfig () {
         /* Opt out of parallel tests on CI. */
         workers: '50%',
         /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-        reporter: [['html', { outputFolder: 'lit-visual-report' }]],
+        reporter: [['html', { outputFolder: '../../../lit-visual-report' }]],
```

Comment:
Probably it works right, but seems to have one extra `../`. If not, please dismiss the comment.

Reply:
Fair point. I've taken a look at this and checked in the actions tab and I'm pretty sure that what I've got is correct as I can see the artifacts have been produced in the correct place.